test/parallel/test-stream-prototype-property.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+require('../common');
+
+// This test ensure state has initialized when accessing property
+// from prototype.
+// Make sure it wouldn't throw TypeError when accessing
+// readableLength, readableHighWaterMark from
+// stream.Readable.prototype. Same for `Duplex` and `Writable`.
+
+const { Duplex, Readable, Writable } = require('stream');
+const assert = require('assert');
+
+{
+  function MyDuplex() {
+    assert.strictEqual(this.readableHighWaterMark, undefined);
+    assert.strictEqual(this.readableFlowing, false);
+    assert.strictEqual(this.readableLength, 0);
+    assert.strictEqual(this.writableHighWaterMark, undefined);
+    assert.strictEqual(this.writableLength, 0);
+    Duplex.call(this);
```

Comment:
The call to super should always be done before accessing `this`.

Reply:
Honestly speaking, I learned this from `destroyed` method tests:

https://github.com/nodejs/node/blob/53e0f632dbed27f0889322bcf0244c886c549835/lib/_stream_readable.js#L182-L204

https://github.com/nodejs/node/blob/53e0f632dbed27f0889322bcf0244c886c549835/test/parallel/test-stream-readable-destroy.js#L155-L166

Reply:
@BridgeAR So should I update the test ? Any advice would be appreciated.